examples/blog/blog.js

```diff
@@ -0,0 +1,52 @@
+const rdf = { ...require('@rdfjs/data-model'), ...require('@rdfjs/dataset') }
+const generateIri = require('./lib/generateIri')
+const ns = require('./lib/namespaces')
+const rebase = require('./lib/rebase')
+const unsetGraph = require('./lib/unsetGraph')
+const validate = require('./lib/validate')
+
+async function get (req, res) {
+  if (req.dataset) {
+    const filters = await req.dataset()
+
+    const fromQuad = [...filters.match(null, ns.schema.from, null, null)][0]
+    const from = fromQuad && fromQuad.object
+
+    const toQuad = [...filters.match(null, ns.schema.to, null, null)][0]
+    const to = toQuad && toQuad.object
+
+    if (from || to) {
+      console.log(`filter: ${from && from.value} - ${to && to.value}`)
+    }
+  }
+
+  res.dataset(unsetGraph(req.hydra.resource.dataset))
```

Comment:
I wonder if this should even be necessary?

How about `req.hydra.resource.dataset` is already flattened to the default graph?